rclcpp/src/rclcpp/time_source.cpp

```diff
@@ -260,6 +278,7 @@ void TimeSource::destroy_clock_sub()
 
 void TimeSource::on_parameter_event(const rcl_interfaces::msg::ParameterEvent::SharedPtr event)
 {
+  RCLCPP_DEBUG(logger_, "this node should not get others' parameter event if CFT is supported");
```

Comment:
if CFT is not supported, this prints every single time on parameter event happens. probably we would want to print this message if CFT is supported only?

Reply:
How about
```?
  if (parameter_subscription_->is_cft_supported()) {
    if (event->node != node_base_->get_fully_qualified_name()) {
      RCLCPP_WARN(logger_, "this node '%s' should not get parameter event of another node '%s'",
        node_base_->get_fully_qualified_name(), event->node.c_str());
    }
  }
```

Reply:
OK, I'll update it.